x-pack/plugins/fleet/server/services/package_policies/experimental_datastream_features.ts

```diff
@@ -126,10 +135,16 @@ export async function handleExperimentalDatastreamFeatureOptIn({
 
     let sourceModeSettings = {};
 
+    const indexTemplateRes = await esClient.indices.getIndexTemplate({
+      name: featureMapEntry.data_stream,
+    });
+
     if (isSyntheticSourceOptInChanged) {
       sourceModeSettings = {
         _source: {
-          ...(featureMapEntry.features.synthetic_source ? { mode: 'synthetic' } : {}),
+          ...(featureMapEntry.features.synthetic_source
+            ? { mode: 'synthetic' }
+            : { mode: 'stored' }),
```

Comment:
`stored` mode was just removed [here](https://github.com/elastic/kibana/pull/149209), how come we add it back?

Reply:
some local changes that I forget to clean, I will fix that